test/functionalTest/cases/3389_forced_update_option/3389_forced_update_option_number.test

```diff
@@ -0,0 +1,495 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Force update for NGSIv2 POST /v2/entities, POST /v2/subscriptions, Dump and reset accumulator, POST /v2/entities/Room1?options=forcedUpdate, Dump and reset accumulator, POST /v2/entities/Room1?options=append,forcedUpdate, Dump and reset accumulator, PUT /v2/entities/Room1?options=forcedUpdate, Dump and reset accumulator,PUT /v2/entities/Room1/attrs/pressure?options=forcedUpdate, Dump and reset accumulator, PUT /v2/entities/Room1/attrs/pressure/value?options=forcedUpdate, Dump and reset accumulator, PATCH /v2/entities/Room1?options=forcedUpdate, Dump and reset accumulator.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, to create Room1 with temperature and pressure.
+# 02. POST /v2/subscriptions, to create subscription.
+# 03. Dump and reset accumulator, see notification.
+# 04. POST /v2/entities/Room1?options=forcedUpdate, update entity.
```

Comment:
...update entity -> ...update entity with same values of temperature and pressure.

In addition, as in the other test, the following cases are missing:


* POST /v2/op/update, with actionType "append"
* POST /v2/op/update, with actionType "update"

Reply:
Fixed in 43339f5